backend/icpy/api/rest_api.py

```
@@ -596,6 +643,7 @@ async def get_file_raw(path: str):
                         break
 
                 if abs_path is not None:
+                    logger.info(f"[REST][raw] Using local streaming path abs={abs_path}")
                     # Security: ensure selected path is inside either fs_root or workspace_root
                     if not (abs_path.startswith(fs_root) or abs_path.startswith(workspace_root)):
                         raise HTTPException(status_code=400, detail="Path outside workspace root")
```

Ruff (0.14.0)

Abstract `raise` to an inner function

(TRY301)